manual/src/html_processing/scss/manual.scss

```diff
@@ -280,10 +283,17 @@ blockquote.quote{
     color:#000;
     &::before{@include diamond}
 }
+.center {
+	text-align: center;
```

Comment:
Horizontally aligning text on the center seems like a mistake outside of tables?

Reply:
It is necessary to center the displayed syntax fragments, as in the screenshot. These rules are just copied from the old HTML manual CSS.

Reply:
It seems to me that having some indenting would be better than centering texts.